datalab/settings.py

```diff
@@ -37,6 +37,7 @@ def get_list_from_env(variable, default=None):
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
+TMP_DIR = os.getenv('TMP_DIR', BASE_DIR / 'tmp')
```

Comment:
I think we want to call this something more descriptive - perhaps TEMP_FITS_FILE_CACHE